submissions/madmaxWMFU/frogger-game/js/app.js

```diff
@@ -0,0 +1,95 @@
+const playerStartValue = {
+    "x": 200,
+    "y": 400,
+    "icon": "images/char-boy.png"
+};
+
+let score = document.querySelector(".user-score");
+
+class Enemy {
+    constructor(x, y, speed) {
+        this.x = x;
+        this.y = y;
+        this.sprite = "images/enemy-bug.png";
+        this.speed = speed;
+        this.player = player;
```

Comment:
`player` property looks like a global. You pass it to constructor, but not access them.